tests/__compound-components__/Filters/Facets.tsx

```diff
@@ -1,8 +1,9 @@
 import { Filters } from '../../../src/components';
 
-export function Facets({ searchOnChange = true }: { searchOnChange?: boolean }): JSX.Element {
+export function Facets(args: Filters.FacetsProps): JSX.Element {
```

Comment:
For the pure Jest tests, do we need to modify the args of the the top-level function? I understand why we need to do this for the Stories, that's how we get controls working properly.

Reply:
These top-level function components are used in their corresponding Stories, so in order to set the controls for those Stories, the args need to be passed in